maps/python/quathelpers.py

```diff
@@ -78,6 +78,9 @@ def AddTimingToPointingQuats(fr, key, timing_ref='RawBoresightAz'):
     if key not in fr:
         return
 
+    if isinstance(fr[key], G3TimestreamQuat):
+        return
+
```

Comment:
This is a good change, but maybe should go in separately to this PR?

Reply:
I added it here since this shim module is only necessary in conjunction with the mock observing module...